.github/workflows/docker-image-pruning.yml

```diff
@@ -0,0 +1,36 @@
+# .github/workflows/prune-temp-ghcr-images.yml
+
+name: Docker Image Pruning for GHCR
```

Comment:
What is an example of an image to be pruned?

Reply:
@davinchia - Images to be pruned would be the temporary/test images from the other workflow, which will all be prefixed with "draft-pr-"

Here's a link to the DHCR registry for `java-connector-base`: https://github.com/airbytehq/airbyte/pkgs/container/java-connector-base

And a full list of generated container packages is here:

- https://github.com/orgs/airbytehq/packages

Reply:
I was mostly asking to understand; I agree it's good practice to clean these up.

I also like the `draft-pr` naming convention. Probably can be shortened to `pr`, however no a strong opinion.